### ISSUE TITLE
Why use Google Analytics?

### ISSUE THREAD

**Issue by pquerner** [hharnisc/tabbie#17]
See title. Why is this in effect? Whats the use of it here?

**Comment by hharnisc** [comment]
Hey @pquerner, great question!

Here's why I'm using Google Analytics:

- Get a better understanding of how people are using the app

Example: The average number of tabs that people save is around 5 tabs. This gives me a better idea of how to design screens that can display what's in a tab group. The code to collect that is here: https://github.com/hharnisc/tabbie/blob/master/popup.js#L181-L186

I don't collect any urls or any other personally identifiable information. 

- Determine if there are any features that can be cut

Example: The "save and close" button is used about 13% of the time, which either means it's too hidden or not useful. It's taking up a fair amount of screen real estate, so it could be a candidate to be removed. Hard to tell, but something to watch. The code to collect that is here: https://github.com/hharnisc/tabbie/blob/master/popup.js#L214-L219

Same thing, no urls and no personally identifiable information

- Determine the next language to translate the app (i18n)

Only a little more than 65% of the users of this app have English (us + gb) set as their language code. The next most common is Spanish and then Chinese (Taiwan). The code to collect that is here: https://github.com/hharnisc/tabbie/blob/master/analytics.js#L10


So to summarize *why* I'm interested in patterns that emerge from the group, and not so much the individual. I want to make sure that I spend my time in the right places using the data, rather than making guesses based on intuition.

**Comment by pquerner** [comment]
Thank you for being so open to us @hharnisc .
I for one can live with your solid explanation.

**Comment by hharnisc** [comment]
Thank you for asking about this, it's a very important question! Working on adding a note in the readme that makes it clear that Google analytics is used and what events are connected. Transparency is important to me because I believe it builds trust.
